addon/lib/Feature.jsm

```diff
@@ -932,6 +1009,25 @@ class Feature {
           this.introPanelMessages[this.treatment],
           isIntroPanel
         );
+
+        let counter = this.treatment === "private" ?
+        this.state.blockedResources.get(win.gBrowser.selectedBrowser) :
+        this.state.timeSaved.get(win.gBrowser.selectedBrowser);
+
+        // log page action click
+        Storage.get("behavior-summary").then((behaviorSummary) => {
```

Comment:
Not sure if this should be `async` + `await`ed to be consistent with other usages, or if not worth the effort.

Reply:
Not a blocker; but I agree consistency is a lofty ideal to aspire to.

Reply:
I did a quick Ctrl+F on the PR files and found 4 `.then(` strings, and 0 `.catch(` strings. I can individually comment on each line if that in some way helps in any possible efforts to `await` our `async`s. 🤷‍♀️ 

Otherwise, I'm fine with merging without those changes and I can file a separate bug to async-ify any lurking promises.

Reply:
Let's file a separate bug for that if you don't mind!